workers/generic-worker/errorreport/errorreport.go

```diff
@@ -0,0 +1,28 @@
+package errorreport
+
+import (
+	"github.com/taskcluster/taskcluster/v30/internal/workerproto"
+)
+
+func Send(proto *workerproto.Protocol, err error, debugInfo map[string]string) {
+	title := "generic-worker error"
+	description := err.Error()
+	// could support differentiating for panics
+	kind := "worker-error"
+	// convert debugInfo from map[string]string to map[string]interface{}
+	extra := map[string]interface{}{}
+	for k, v := range debugInfo {
+		extra[k] = v
```

Comment:
Note, some of the fields may have empty values (e.g. `instanceType` is an optional config setting) so some of the properties may have empty values. Does worker manager handle this graciously, or should we remove properties with empty values here? I suspect keeping them is better (as you have done), so that it is explicit that the value was unset, rather than just not included, but we should definitely decide if this is what we want.